source/codegen/metadata/nixnet/custom_proto.mako

```diff
@@ -145,4 +145,19 @@ message DbGetDatabaseListResponse {
     string alias_buffer = 2;
     string file_path_buffer = 3;
     uint32 number_of_databases = 4;
+}
+
+message DbGetDBCAttributeRequest {
```

Comment:
Why do we need to implement these messages in custom proto? Looks like we should be able to auto generate these.

Reply:
Done using CustomCode codegen_method for this API